impress-org/givewp
# Feature: Add Pro tip message to Legacy Form Editor

## Description

This PR includes a dismissible "Pro tip" recommendation for recurring within the form editor on the "Donation Options" tab. The recommendation should be visible unless the user has the add-on active and/or has previously dismissed it as an option.

## Affects

Form Editor

## Visuals



## Testing Instructions

- Navigate to the Donation forms page
- Select "EdIt" on any form
- Under Donation Form Options -> Select the "Donation Options" tab
- View recommendation
- Activate Recurring addon
- Verify the recommendation is no longer displayed
- Deactivate the add on and revisit the page
- Dismiss the Option
- Refresh the page verify the recommendation remains closed

## Pre-review Checklist
